tests/testthat/test-robustbayesianmetaanalysis.R

```diff
@@ -24,33 +24,33 @@ if (FALSE){
 }
 
 # path to the pre-fitted RoBMA model
-fitted_path <- file.path("robmaFit.RDS")
+fittedPath <- file.path("robmaFit.RDS")
 
 ### prior distibutions plots 
 {
   options <- jaspTools::analysisOptions("RobustBayesianMetaAnalysis")
-  options$.meta <- list(input_CI = list(containsColumn = TRUE), input_ES = list(
-    containsColumn = TRUE), input_N = list(containsColumn = TRUE), 
-    input_N1 = list(containsColumn = TRUE), input_N2 = list(containsColumn = TRUE), 
-    input_SE = list(containsColumn = TRUE), input_labels = list(
-      containsColumn = TRUE), input_t = list(containsColumn = TRUE), 
-    priors_mu = list(list(name = list(containsColumn = TRUE)), 
+  options$.meta <- list(inputCI = list(containsColumn = TRUE), inputSE = list(
```

Comment:
.meta? Is there a specific reason why there's a dot in the name?

Reply:
this is a remnant from generating unit tests using the jaspTools package, as far as I understand, they should be safe to delete, will do that